skiko/src/jsMain/kotlin/org/jetbrains/skiko/SkiaLayer.js.kt

```diff
@@ -11,7 +12,7 @@ actual open class SkiaLayer {
 
     actual var renderApi: GraphicsApi = GraphicsApi.WEBGL
     actual val contentScale: Float
-        get() = 1.0f
+        get() = window.devicePixelRatio.toFloat()
```

Comment:
not a big deal but may be we should just keep Double and not convert till it needed? contentScale is use in expression after all.